src/coreclr/inc/daccess.h

```diff
@@ -2365,6 +2368,23 @@ inline type* DacUnsafeMarshalSingleElement( ArrayDPTR(type) arrayPtr )
 //
 //----------------------------------------------------------------------------
 
+typedef DPTR(size_t)  PTR_size_t;
+#ifdef NATIVEAOT
+typedef DPTR(int8_t)          PTR_Int8;
```

Comment:
We should rename and unify these with VM. We do not have `Int8` type in native AOT, so it does not make sense to have `PTR_Int8`. I think that I have missed renaming these when renaming Redhawk special primitive type definitions to standard C++ types.

We should expose the types that work in native AOT unconditionally to have common set of types to use.

.... common ones ...

#ifndef NATIVEAOT
... special ones ...
#endif
```